test/regressions/index.js

```diff
@@ -99,18 +99,18 @@ const blacklist = [
   'docs-components-transitions', // Needs interaction
   'docs-components-tree-view/ControlledTreeView.png', // Redundant
   'docs-components-tree-view/CustomizedTreeView.png', // Flaky
-  'docs-components-use-media-query', // Not needed
-  'docs-customization-breakpoints', // Not needed
-  'docs-customization-color', // Not needed
+  'docs-components-use-media-query',
+  'docs-customization-breakpoints',
+  'docs-customization-color',
   'docs-customization-default-theme', // Redux isolation
   'docs-customization-density/DensityTool.png', // Redux isolation
-  'docs-customization-typography/ResponsiveFontSizesChart.png', // Not needed
-  'docs-discover-more-languages', // Not needed
-  'docs-discover-more-showcase', // Not needed
-  'docs-discover-more-team', // Not needed
-  'docs-getting-started-templates', // Not needed
+  'docs-customization-typography/ResponsiveFontSizesChart.png',
+  'docs-discover-more-languages',
+  'docs-discover-more-showcase',
+  'docs-discover-more-team',
+  'docs-getting-started-templates',
   'docs-getting-started-templates-album/Album.png', // Flaky image loading
-  'docs-getting-started-templates-blog', // Not needed
```

Comment:
`// Not needed` Maybe we should rename it `// Nothing to test`? Basically, these pages are supposed to yield close to no value or a low ROI.

Reply:
Why is there nothing to test? They're not empty as far as I know.

Reply:
In this case, I imagine the images make it flaky. So the "label" isn't correct.

Reply:
Feel free to add any descriptive label you have. The current ones I removed where classic "comments for comments sake"-noise.

Reply:
Ok, cool, let's add more specific comments. They should add information. "No needed" is equivalent to having the tets inside the blacklist array: noise.

Reply:
Perfect. That helps a lot. Is this good to merge in your opinion? Would be nice to get this in before #20187